src/servers/src/http/prom_store.rs

```diff
@@ -30,9 +34,14 @@ use snafu::prelude::*;
 
 use crate::error::{self, Result, UnexpectedPhysicalTableSnafu};
 use crate::prom_store::snappy_decompress;
+use crate::proto::PromWriteRequest;
 use crate::query_handler::{PromStoreProtocolHandlerRef, PromStoreResponse};
 
 pub const PHYSICAL_TABLE_PARAM: &str = "physical_table";
+lazy_static! {
+    static ref PROM_WRITE_REQUEST_POOL: Pool<PromWriteRequest> =
+        Pool::new(256, PromWriteRequest::default);
```

Comment:
I am curious if this default size is the best choice.

Reply:
> I am curious if this default size is the best choice.

In vmbench, each `PromWriteRequest` containing 10k timeseries costs 10ms to decode, thus we expect no more 100 requests every second. thus we relax the pool size to 256.